OpenSim/Examples/ExampleHopperDevice/helperMethods.h

```diff
@@ -184,7 +195,7 @@ inline void simulate(Model& model, SimTK::State& state, bool saveStatesFile)
 
     // Simulate until the user presses ESC (or enters 'q' if visualization has
     // been disabled).
-    while (true) {
+    while (!simulateOnce) {
```

Comment:
I intended to change `while` to `do-while`. 

The current version works without visualizer with one simulation. It is broken if visualizer is turned on.